packages/gestalt/src/contexts/GlobalEventsHandlerProvider.js

```diff
@@ -0,0 +1,40 @@
+// @flow strict
+import { createContext, type Context, type Element, type Node, useContext } from 'react';
+
+export type NoopType = () => void;
+
+type SheetMobileHandlersType = {|
```

Comment:
nit: we're not exporting this type or using it elsewhere in this file, so I'd just inline it in GlobalEventsHandlerContextType